File: paas/slam-em-pm-integrator/src/main/java/es/bsc/paas/EmslamintegratorApplication.java
package es.bsc.paas;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.jms.annotation.EnableJms;
import org.springframework.scheduling.annotation.EnableScheduling;

/**
 * This class just defines the application entry point
 */
@SpringBootApplication
@EnableAutoConfiguration
@EnableJms
@EnableScheduling
public class EmslamintegratorApplication {
    public static final void main(String[] args) {
        SpringApplication.run(EmslamintegratorApplication.class, args);
    }
}

File: paas/emslamintegrator/src/main/kotlin/es/bsc/paas/EmslamintegratorApplication.kt
package es.bsc.paas

import org.apache.activemq.broker.region.Queue
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.EnableAutoConfiguration
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.context.annotation.Bean
import org.springframework.jms.annotation.EnableJms
import org.springframework.jms.config.DefaultJmsListenerContainerFactory

@SpringBootApplication
@EnableAutoConfiguration
@EnableJms
open class EmslamintegratorApplication {

    companion object {
        @JvmStatic fun main(args: Array<String>) {
            SpringApplication.run(EmslamintegratorApplication::class.java, *args)
        }
    }
}

